src/UI/Art/SpriteArtist/Sprite.java

```diff
@@ -63,9 +63,12 @@ public Sprite(BufferedImage spriteSheet, int spriteWidthPx, int spriteHeightPx)
 
   /**
    * Sprite copy-constructor. Perform a deep-copy on each of the other sprite's frames.
-   * @param other
    */
   public Sprite(Sprite other)
+  {
+    this(other, false);
+  }
+  public Sprite(Sprite other, boolean hFlip)
```

Comment:
Why create a flipped copy instead of just drawing the image flipped (as is currently done for the idle sprites)?

Reply:
Because I didn't want to put code in to figure out which direction of sprite (EAST/WEST) we have and draw that flipped in the animation code.
Much simpler to just make sure we have all relevant directions and just use the right one.

Reply:
I mean. We already do that for IDLE.

Reply:
And the circumstances are different, because we don't know at `UnitSpriteSet` creation time which way IDLE should face. We *do* know for movement.

Reply:
By that logic, we should just create idleEast and idleWest copies of the sprite.

Reply:
That's not a terrible idea, though I feel like it would necessitate some code changes that don't really need to be made.